helm-chart-sources/pulsar/ci/test-notls-values.yaml

```diff
@@ -108,6 +108,8 @@ grafanaDashboards:
 
 pulsarAdminConsole:
   replicaCount: 1
+  ingress:
+    enabled: true
```

Comment:
I addressed this in #162 by changing the service to a `ClusterIP`. I think the underlying issue is with MetalLB, but I'm not sure we even need to deploy that daemonset in our tests. There is no need to test k8s behavior for load balancers in our tests.